src/pages/components/tabs/style.mdx

```diff
@@ -33,17 +166,73 @@ Tab label should be set in sentence case, and should not exceed three words.
+| --------------- | --------------------------- | -------- | ------------- |
+| Tab             | border-bottom               | 2px      | –             |
+| Label           | padding-left, padding-right | 16 / 1   | `$spacing-05` |
+|                 | padding-top, padding-bottom | 8 / 0.5  | `$spacing-03` |
+| Icon            | padding-right               | 16 / 1   | `$spacing-05` |
+|                 | padding-left                | 8 / 0.5  | `$spacing-03` |
+|                 | svg                         | 16 x 16  | —             |
+| Tab             | margin-left                 | 1px      | —             |
+| Icon-only       | svg                         | 16 x 16  | -             |
+|                 | svg                         | 20 x 20  | —             |
+| Scrollable icon | svg                         | 16 x 16  | -             |
+
+<Row>
+<Column colLg={8}>
+
+![Structure and spacing measurements for line tabs in px and rem](images/tab-style-7.png)
+
+</Column>
+</Row>
+
+<Caption>Structure and spacing measurements for line tabs | px / rem</Caption>
+
+### Contained tabs
+
+| Element               | Property                    | px / rem | Spacing token |
 | --------------------- | --------------------------- | -------- | ------------- |
-| `.bx--tabs__nav-item` | border-bottom               | 2px      | –             |
-| `.bx--tabs__nav-link` | padding-left, padding-right | 16 / 1   | `$spacing-05` |
-| `.bx--tabs__nav-link` | padding-top, padding-bottom | 8 / 0.5  | `$spacing-03` |
-| `bx--tabs__nav-item`  | margin-left                 | 1px      | -             |
+| Tab                   | border-top                  | 2px      | –             |
+| Label                 | padding-left, padding-right | 16 / 1   | `$spacing-05` |
+| Icon                  | padding-right               | 16 / 1   | `$spacing-05` |
+|                       | padding-left                | 16 / 1   | `$spacing-05` |
+|                       | svg                         | 16 x 16  | —             |
+| Tab                   | border-right                | 1px      | —             |
+| Icon-only             | svg                         | 20 x 20  | -             |
+| Scrollable icon       | svg                         | 16 x 16  | —             |
+| Scrollable background | border-right, border-left   | 1px      | -             |
+
+<Row>
+<Column colLg={8}>
+
+![Structure and spacing measurements for contained tabs in px and rem](images/tab-style-8.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Structure and spacing measurements for contained tabs | px / rem
+</Caption>
+
+## Sizes
+
+| Element                 | Property | Height (px / rem) |
+| ----------------------- | -------- | ----------------- |
+| Line tab                | Medium   | 40 / 2.5          |
+| Icon-only line tab      | Medium   | 40 / 2.5          |
+|                         | Large    | 48 / 3            |
+| Contained tab           | Large    | 48 / 3            |
```

Comment:
I think since contained tabs only has one size (large) I would take it out of the size table and just include that information as `height` its structure section. I'd then make the heading "size" and h4 under the contained tab section.

It then might help to add a sentence about why line tab icon only has a size variation but the normal line tab doesn't. I think this section is more confusing then it needs to be.